ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTest.java

```diff
@@ -2018,6 +2015,12 @@ public void ethGetStorageAtBlockNumber() throws Exception {
 
   @Test
   public void ethGetStorageAtInvalidParameterStorageIndex() throws Exception {
+    when(blockchainQueries.getBlockchain()).thenReturn(blockchain);
```

Comment:
Same as in the [comment above](https://github.com/hyperledger/besu/pull/4807/files#r1091801204).